src/common/protocol_client/master_chunk_server_manager_service_client.cc

```diff
@@ -0,0 +1,39 @@
+#include "src/common/protocol_client/master_chunk_server_manager_service_client.h"
+
+#include <memory>
+
+#include "google/protobuf/stubs/status.h"
+#include "src/common/utils.h"
+
+using gfs::common::utils::ConvertGrpcStatusToProtobufStatus;
+using google::protobuf::util::Status;
+using google::protobuf::util::StatusOr;
+using grpc::ClientContext;
+using protos::grpc::ReportChunkServerReply;
+using protos::grpc::ReportChunkServerRequest;
+
+namespace gfs {
+namespace service {
+
+StatusOr<ReportChunkServerReply>
+MasterChunkServerManagerServiceClient::SendRequest(
+    const ReportChunkServerRequest& request, ClientContext& context) {
+  ReportChunkServerReply reply;
+  grpc::Status status = stub_->ReportChunkServer(&context, request, &reply);
+
+  if (status.ok()) {
```

Comment:
can you help me make a TODO here to update it to a new method `ReturnStatusOrFromGrpcStatus` that will be introduced by pull request #25 to simply this code

Reply:
done